src/driver.cpp

```diff
@@ -18,8 +19,20 @@ int driver::parse_file(const std::string &filename) {
 	return parse_string(buffer.str());
 }
 
+std::string driver::import_files(const std::string &in) {
```

Comment:
This function does a lot of things. 
Maybe split it up?

Reply:
How would you suggest doing that? It seems pretty small already.